openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -324,6 +324,20 @@ def display_name_with_default(self):
         """
         return course_metadata_utils.display_name_with_default(self)
 
+    @property
+    def display_name_with_default_escaped(self):
```

Comment:
@nasthagiri Does this require a bump in the version?  I don't think so, but I would like you to confirm.

Two changes were made:
1. This new property was added (display_name_with_default_escaped).  It will ultimately be deleted once it is no longer in use.  When it is deleted, will the version need to change?  
2. The older display_name_with_default no longer escapes.  This doesn't make it to the db, so I'm guessing this doesn't require a version change, but I do not know for sure.

Are these objects, including the properties, serialized and cached anywhere?

Thank you!